pybossa/forms/dynamic_forms.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, *args, **kwargs):
     if data_access_levels:
         ProjectFormExtraInputs.amp_store = BooleanField(
             lazy_gettext('Opt in to store annotations on Annotation Management Platform'),
-            render_kw = {'checked': obj and obj.amp_store})
```

Comment:
Wasn't the original correct? :) If `amp_store == True` check the box. If `amp_store == False` uncheck the box.

```python
render_kw = {'checked': obj and obj.amp_store})
```